### ISSUE TITLE
CMAK on the latest apache kafka release?

### ISSUE THREAD

**Issue by JoshuaBunag** [yahoo/CMAK#838]
Will this still work on the latest apache kafka version 2.8.0? and will this be maintained or is it a dead project?

I'm planning to use this for my mini project, if not what else can you recommend for managing clusters and brokers and more?

Thanks!

**Comment by infordb** [comment]
I'm curious about that, too.
I try to use cmak for kafka2.7.0, but it seems that kafka 2.7.0 is not currently supported.
Do you have any plans to support the latest version of kafka?

**Comment by ialejandro** [comment]
Yes, it works @JoshuaBunag @infordb but... exists a bug that shows you only 2.4.0 zk version. But, I checked the rest of functions and work properly (by now...) :+1:

**Comment by MaximGurschi** [comment]
@ialejandro Do you manage to see anything in "consumer" tab when plugin with Kafka 2.8.0?

**Comment by JoshuaBunag** [comment]
> @ialejandro Do you manage to see anything in "consumer" tab when plugin with Kafka 2.8.0?

i tried it with the latest release for kafka and it does not show anything, so it's broken

**Comment by fanruinet** [comment]
There's a pending pull request for Kafka 2.8.0: https://github.com/yahoo/CMAK/pull/842
It's not passing the CI build. I built one from that branch and it works well.

**Comment by MrMunki** [comment]
Any update on this release?

**Comment by JoshuaBunag** [comment]
@MrMunki nope, nothing from the devs as well. this might be a dead project already.

**Comment by danieltoader** [comment]
Hello, I made a simple PR to add latest kafka versions to CMAK here: https://github.com/yahoo/CMAK/pull/871 i managed to build a docker image from the sbt generated files and everything works ok (I tested it with 2.8.1)
